torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2700,6 +2700,37 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad, **kwargs)
     )
     return [SampleInput(tensor) for tensor in tensors]
 
+def sample_inputs_fmod_remainder(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, dtype=dtype, device=device, requires_grad=requires_grad)
+
+    cases = (
+        ((S, S, S), (), 1.5, False),
+        ((), 1.5, (), False),
+        ((S, S, S), (S, S, S), 1.5, False)
+    )
+
+    # Sample inputs with broadcasting
+    cases_with_broadcasting = (
+        ((S,), (S, S, S), 1.5, True),
+        ((S, S, S), (S,), 1.5, True),
+        ((S, 1, S), (S, S), 1.5, True),
+        ((), (S, S, S), 1.5, True),
```

Comment:
I think you are missing the following case:

https://github.com/pytorch/pytorch/blob/6997e7bd3934f9d8fa3d425336f2e20de2705358/torch/testing/_internal/common_methods_invocations.py#L6285

Reply:
Yes, thanks for pointing it out. This has been added now.